modules/claims_api/app/workers/claims_api/ews_updater.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+require 'bgs'
+require 'claims_api/claim_logger'
+
+module ClaimsApi
+  class EwsUpdater
+    include Sidekiq::Worker
+
+    def perform(bgs_claim, target_veteran, ews)
```

Comment:
how will we resubmit with the `bgs_claim` here? for example, the script looks something like
```
claim_id = 'r34l-gu1d'
claim = ClaimsApi::EvidenceWaiverSubmission.find(claim_id)

ClaimsApi::EvidenceWaiverSubmission.perform_async(claim.id)
```

We'll probably want to do a lookup and check here for updating the BGS claim and pdf.

Reply:
the pdf is another ticket